src/comparator/wrappers.py

```diff
@@ -526,29 +521,19 @@ def name(self):
 
     @property
     def input(self):
-        """Return the shortened input type of a method. We only need the name
-        of the message to query in the messages_map.
-        For example: `.example.v1.FooRequest` -> `FooRequest`
-        """
-        input_type = self.method_pb.input_type.rsplit(".", 1)[-1]
+        """Return the shortened input type of a method."""
         # MethodDescriptorProto.input_type has field number 2
-        return WithLocation(input_type, self.source_code_locations, self.path + (2,))
+        return WithLocation(
+            self.method_pb.input_type, self.source_code_locations, self.path + (2,)
+        )
```

Comment:
We decide to use directly the input name (instead of shortened name) because it is also the key name in the messages map. But this also brings the need to check minor version updates when comparing methods. Same as output type name.